app/controllers/api/users_controller.rb

```diff
@@ -2,18 +2,28 @@ class Api::UsersController < ApiController
   # before_action :authenticate
   before_action :authenticated?
 
+  def index 
+    users = User.all
+    render json: users
+  end
+
   def create
     user = User.new(user_params)
     if user.save
-      render json: user
+      render json: user, status: 201
     else
       render json: { errors: user.errors.full_messages }, status: 422
     end
   end
 
-  def index 
-    users = User.all
-    render json: users
+  def destroy
+    begin
+      user = User.find(params[:id])
+      user.destroy
+      render json: {}, status: 204
+    rescue ActiveRecord::RecordNotFound
+      render json: {}, status: 404
+    end
   end
```

Comment:
Note: I'm a trying to verify whether Rails will actually raise an exception in case the destroy operation couldn't be performed. It may not and it may just return `false`, so in that case, one would have to rely on something like `if user.destroy … else … end`.